aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -1335,7 +1335,7 @@ public void run() {
                         break;
                     case DONE:
                         detectedContinuation = null;
-                        Log.d(TAG, "confirmSignIn called after signIn has succeeded");
+                        callback.onError(new IllegalStateException("confirmSignIn called after signIn has succeeded"));
```

Comment:
This represents a change in behavior, right? We should call it out as a breaking change.